module/VuFind/src/VuFind/ILS/Logic/AvailabilityStatus.php

```diff
@@ -115,6 +119,20 @@ public function getStatusDescription(): string
         }
     }
 
+    /**
+     * Get extra status information.
```

Comment:
I think it might be a good idea to add a little more documentation about the extra status information -- e.g. "Get extra status information (used to provide tokens for translation)" if that's the only known purpose for this.

It's also an open question whether we want to map the `%%` on dynamically here or require them up-front. I don't have a strong opinion about the best way, but whatever we do, we should document the constructor parameters and return values clearly so it doesn't become confusing about what's going in and what's coming out. If we do dynamically add the `%%` characters, maybe we should consider including an argument here to bypass that functionality if somebody wants the raw values.